style.css

```diff
@@ -67,3 +67,277 @@ main {
+  background-color: #fff;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+  margin-top: 20px;
+  padding: 16px;
+  width: 50%;
+}
+
+.card {
+  margin-left: 20px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background: #fff;
+  padding: 16px;
+}
+
+.works {
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  gap: 50px;
+}
+
+.lang-drop {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 30px;
+}
+
+.js {
+  display: flex;
+  align-items: center;
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin-top: 20px;
+}
+
+#js {
+  margin-left: 10px;
+}
+
+.lg {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+.txt {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  margin-left: 10px;
+}
+
+#about-me {
+  margin-left: 20px;
+}
+
+.multi-post {
+  font-family: 'Poppins', sans-serif;
+  font-style: bold;
+  font-size: 32px;
+  line-height: 44px;
+  line-height: 92%;
+  margin-top: 8px;
+}
+
+.about-myself {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  margin-top: 100px;
+  border: 1px solid #dfe1e6;
+  border-top-right-radius: 100px;
+  background: #fff;
+  width: 430px;
+}
+
+.icon {
+  width: 48px;
+  height: 48px;
+  left: 12px;
+}
+
+.side-drop {
+  display: flex;
+  justify-content: space-between;
+  border-radius: 0;
+  border-color: #dfe1e6;
+  border-width: 0 0 2px 0;
+  border-style: none none solid none;
+  margin-top: 30px;
+}
+
+.resume {
+  margin-top: -200px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+  padding: 16px;
+  width: 50%;
+  background-color: #fff;
+}
+
+.me {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: 100px;
+}
+
+.hello {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  margin-top: 200px;
+  gap: 20px;
+  background-image: url("./images/header-bg.png");
+  text-align: center;
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: auto;
+  border: 2px solid white;
+  border-radius: 100px 0 0 0;
+  height: 779px;
+  width: 400px;
+  margin-left: -20px;
+}
+
+.contact {
+  margin-top: 100px;
+}
+
+.input {
+  height: 170px;
+  width: 360px;
+  left: 0;
+  top: 0;
+  border-radius: 8px;
+}
+
+.ipsum {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+.get-in {
+  margin-top: 20px;
+  height: 48px;
+  width: 142px;
+  border-radius: 8px;
+  padding: 12px 16px 12px 16px;
+}
+
+button:hover {
+  background-color: #1e90ff;
+}
```

Comment:
- [ ] Great job, @gitabiy :clap: Just as the previous reviewer highlighted [here](https://github.com/gitabiy/Portfolio-setup-and-mobile-first/pull/2#discussion_r1148423615), kindly use the appropriate color values for your button interactions.
- To make it easier for you, the correct CSS values are:
```
button:hover {
  background-color: #6070ff;
  color: #4053fc:
}

button:active {
  background-color: #2230d2;
}
```
Using this will make you fulfill all requirements, so you can proceed.

Reply:
Hi @AbayomiOlaoye ,
Can you please tell me where you came up with the color codes for button hover and active from figma?
And thank you for your positive review.

Reply:
Hi @AbayomiOlaoye, thank you once again I have managed to see it.